packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/CozyMarkerBuilder.java

```diff
@@ -57,11 +58,16 @@ private Path addPointerOnMarkerCenter(Bitmap marker, int pointerWidth, int point
         return pointer;
     }
 
-    private Bitmap getIconBitmap(String svgIcon, int width, int height) {
-        if(svgIcon == null)
+    private Bitmap getIconBitmap(String svgIcon, int width, int height, int rgbColor) {
+        if (svgIcon == null)
             return null;
         
-        final Bitmap bitmap = markerCache.getBitmapFromMemCache(svgIcon);   
+        String key = "" + svgIcon.hashCode() + width + height + rgbColor;
```

Comment:
not a necessary suggestion, but would something like this work?
```suggestion
        String key = String.format("%d%d%d%d", svgIcon.hashCode(), width, height, rgbColor);
```

Reply:
Thanks!

Reply:
Applied!